tools/src/launcher/Main.hs

```diff
@@ -322,16 +311,25 @@ writeWindowsUpdaterRunner runnerPath = do
 
 spawnNode
     :: (FilePath, [Text], Maybe FilePath)
-    -> Shell (ProcessHandle, Async ExitCode, FilePath)
+    -> IO (ProcessHandle, Async ExitCode, FilePath)
 spawnNode (path, args, mbLogPath) = do
-    echo "Starting the node"
+    putStrLn "Starting the node"
+    -- We don't explicitly close the `logHandle` here,
+    -- but this will be done when we run the `CreateProcess` built
+    -- by proc later is `system'`:
+    -- http://hackage.haskell.org/package/process-1.6.1.0/docs/System-Process.html#v:createProcess
     (logPath, logHandle) <- case mbLogPath of
         Just lp -> do
-            mktree (FP.directory lp)
-            (lp,) <$> appendonly lp
+            createDirectoryIfMissing True (directory lp)
+            (lp,) <$> openFile lp AppendMode
         Nothing -> do
             tempdir <- liftIO (fromString <$> getTemporaryDirectory)
-            mktemp tempdir "cardano-node-output.log"
+            -- FIXME (adinapoli): `Shell` from `turtle` was giving us no-resource-leak guarantees
```

Comment:
Oh totally, I was indeed thinking about the possibility of inspecting the log _after_ the process exited, that, as you noticed down below, seems useful 😉 

>  So, let's keep the output log, okay. (And later on, if users start complaining, we can just add something to the launcher to go through temp and remove old cardano-node-output.log files.)

Yep, we could later on do a simply check and if we see, say, more than 30 logs which satisfies `"cardano-node-output.log" `T.isInfixOf` x` we can sweep them prior to creating the now log file 😉